vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/source/XYZSource.java

```diff
@@ -19,12 +19,12 @@
 import com.vaadin.flow.component.map.configuration.Constants;
 
 /**
- * Abstract base class for map sources loading tiled images from a map service
- * using the XYZ URL format
+ * Map source for loading tiled images from a map service using the XYZ URL
+ * format
  */
-public abstract class XYZSource extends TileImageSource {
+public class XYZSource extends TileImageSource {
 
-    protected XYZSource(BaseOptions<?> options) {
+    public XYZSource(BaseOptions<?> options) {
```

Comment:
Had to make this non-abstract in order to use it in tests. Shoud be OK, the client-side synchronizer is set up, and there isn't anything else that prevents us from exposing this.